mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -388,8 +381,15 @@ def _compute_moc_postprocess(config, streams, variableMap, calendar,
     latAtlantic = mocDictClimo['latAtlantic']['data']
     dLat = latAtlantic - 26.5
     indlat26 = np.where(dLat == np.amin(np.abs(dLat)))
-    nyears = (dictTseries['endYearTseries'] -
-              dictTseries['startYearTseries'] + 1)
+
+    startYear = days_to_datetime(ds.Time.min()).year
+    endDate = days_to_datetime(ds.Time.max())
+    endYear = endDate.year
+    if endDate.month < 12:
+        endYear -= 1
+    nyears = (endYear - startYear + 1)
+
+    Time = np.zeros(nyears*12)
```

Comment:
@milenaveneziani, I don't think this variable `Time` is ever used, and later on you create `Time` as a NetCDF variable.  I would take this out to avoid confusion.